charts/local-kafka/templates/kafka.yaml

```diff
@@ -31,6 +33,15 @@ spec:
         port: 9093
         type: internal
         tls: false
+      - name: external
+        port: 9094
+        type: nodeport
+        tls: false
+        configuration:
```

Comment:
My only suggestion would be that this listener should be probably configurable in the values file... so that if for some reason `NodePort` is blocked, they don't have to create it. Maybe the entire `listeners` key should just move into the values file?

Reply:
i moved just the external listener for now, mainly so if somebody did want to configure it, they don't have to duplicate the entire `listeners` key. `9092` and `9093` seem fairly standard for the tsl and plaintext ports, so i'm guessing that those probably won't be overwritten much? i can move the entire key if that's preferred though.